inst/examples/06-publishing.Rmd

```diff
@@ -81,7 +81,7 @@ If you are starting from the [previous approach](#ghpages), we will not need to
     Version: 0.0.1
     URL: https://github.com/<USER>/<REPO>
     Imports: bookdown, ggplot2
-    Remotes: rstudio/bookdown
+    Remotes: rstudio/gt
```

Comment:
I got a bit confused by this the first time. I'd recommend adding a package that isn't {bookdown} here instead.

Reply:
Doh! Good call. It doesn’t have to be {gt}. I just remembered the first time I saw this way back when confused me.